tests/E2E/test/campaigns/common_scenarios/module.js

```diff
@@ -336,7 +336,7 @@ module.exports = {
     test('should click on "Discover" button', () => client.waitForExistAndClick(ModulePage.discover_amazon_module_button));
     test('should verify it opens the addons Amazon market place product page in a new tab', () => {
       return promise
-        .then(() => client.switchWindow(id))
+        .then(() => client.switchWindow(id,1000))
         .then(() => client.refresh()) /**Adding refreshing page because sometimes is not well opened we have to refresh it before */
         .then(() => client.checkTextValue(ModulePage.module_name, "Amazon Market Place Module", 'contain'))
```

Comment:
It looks like this selector is defined in 
* tests/E2E/test/selectors/BO/addons_catalog_page.js
* tests/E2E/test/selectors/BO/module_page.js

for the exact same target but with different values. Can you please merge this two values and use the value in `addons_catalog_page`

Reply:
Resolved
you should test with lang = en and country = fr